source/DataAccess/Oracle/OracleDDLHelper.ts

```diff
@@ -53,18 +53,20 @@ export default class OracleDDLHelper implements IDDLHelper {
     private async prioritizeTables(validTableNames: string[]): Promise<string[]> {
         const connection: oracledb.IConnection = await this._connectionPool.getConnection();
 
-        const tableConstraintQueries = validTableNames.map((tableOrViewName) => {
-            return `SELECT
+        const tableInClause = validTableNames.map((tableOrViewName) => {
+            return `'${tableOrViewName}'`;
+        })
+        .join(',');
+
+        const tableConstraintQueries = `SELECT
                 cons.TABLE_NAME as child_table,
                 col.TABLE_NAME parent_table
             FROM
                 ALL_CONS_COLUMNS col
                 JOIN ALL_CONSTRAINTS cons
                 ON cons.R_OWNER = col.OWNER
                 AND cons.R_CONSTRAINT_NAME = col.CONSTRAINT_NAME
-            WHERE cons.TABLE_NAME = '${tableOrViewName}'`;
-        })
-        .join('\nUNION ALL\n');
+            WHERE cons.TABLE_NAME in (${tableInClause})`;
```

Comment:
Was probably a premature optimization doing the 'union all' since I doubt we'll ever hit the 1000 item oracle limit for 'in' clauses. Hopefully this makes the whole thing a bit quicker and less intense on resources.